### ISSUE TITLE
Possible undefined property in 1.3.1 package, related to mouseMoveZoom event?

### ISSUE THREAD

**Issue by alexpreynolds** [higlass/higlass#549]
_Commit_: 

#### Steps to reproduce
1. Installing `higlass` package `1.3.1` via `npm install higlass --save` etc.

```
{
  "name": "epilogos-client",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "axios": "^0.18.0",
    "bootstrap": "^4.1.3",
    "higlass": "^1.3.1",
    "higlass-multivec": "^0.2.0-rc.1",
    "jsonpath": "^1.0.0",
    "react": "^16.6.1",
    "react-bootstrap-table-next": "^1.4.2",
    "react-dom": "^16.6.1",
    "react-grid-system": "^4.3.1",
    "react-protected-mailto": "^1.0.3",
    "react-reveal": "^1.2.2",
    "react-scripts": "2.1.1",
    "react-svg-spinner": "^1.0.2",
    "reactstrap": "^6.5.0"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject",
    "deploy": "rsync -avzhe --progress ./build/* /var/www/epilogos-client"
  },
  "eslintConfig": {
    "extends": "react-app"
  },
  "browserslist": [
    ">0.2%",
    "not dead",
    "not ie <= 11",
    "not op_mini all"
  ],
  "devDependencies": {
    "cross-env": "^5.2.0"
  }
}
```

My HiGlass container is instantiated in a manner somewhat like this (leaving out some code for brevity):

```
import React, { Component } from "react";

// higlass
// cf. https://www.npmjs.com/package/higlass
import "higlass/dist/hglib.css";
import { 
  HiGlassComponent, 
  ChromosomeInfo 
} from "higlass";

class Portal extends Component {
  constructor(props) {
    super(props);
    this.hgView = React.createRef();
    ...
  }

  ... (handlers, logic, etc.) ...

  render() {
    return (
      <div className="higlass-content" style={{"height": this.state.hgViewHeight, zIndex: "1"}} onClick={this.onClickHgViewParent} onMouseEnter={this.onMouseEnterHgViewParent} onMouseLeave={this.onMouseLeaveHgViewParent}>
        <HiGlassComponent
            key={this.state.hgViewKey}
            ref={(component) => this.hgView = component}
            options={{ 
              bounded: true,
              horizontalMargin: 0, 
              verticalMargin: 0 
            }}
            viewConfig={this.state.hgViewconf}
            />
      </div>
    )
  }
}

export default Portal;
```

The value of `this.state.hgViewconf` is ultimately pulled from a GET request to http://explore.altius.org/api/v1/viewconfs/?d=d2-rehCKQx-XcQRn8MZMtA

#### Observed behavior
- I get the following error when I mouseover a track:

```
hglib.js:32698 Uncaught TypeError: Cannot read property 'length' of undefined
    at HorizontalGeneAnnotationsTrack.getDataAtPos (hglib.js:32698)
    at HorizontalGeneAnnotationsTrack.mouseMoveZoomHandler (hglib.js:21530)
    at HorizontalGeneAnnotationsTrack.mouseMoveHandler (hglib.js:32715)
    at hglib.js:26366
    at Array.forEach (<anonymous>)
    at Object.publish (hglib.js:26365)
    at HiGlassComponent.mouseMoveHandler (hglib.js:39625)
    at HTMLUnknownElement.callCallback (react-dom.development.js:147)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:196)
    at invokeGuardedCallback (react-dom.development.js:250)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:265)
    at executeDispatch (react-dom.development.js:571)
    at executeDispatchesInOrder (react-dom.development.js:596)
    at executeDispatchesAndRelease (react-dom.development.js:695)
    at executeDispatchesAndReleaseTopLevel (react-dom.development.js:704)
    at forEachAccumulated (react-dom.development.js:676)
    at runEventsInBatch (react-dom.development.js:844)
    at runExtractedEventsInBatch (react-dom.development.js:852)
    at handleTopLevel (react-dom.development.js:5025)
    at batchedUpdates$1 (react-dom.development.js:19904)
    at batchedUpdates (react-dom.development.js:2246)
    at dispatchEvent (react-dom.development.js:5105)
```

If it helps to see this live: http://epilogos.altius.org:3000/

However, the code for this link is being changed constantly and so I can't guarantee this will behave consistently for the purposes of exploring this issue. I will try to set up a test host, but I apologize as I am up against some deadlines and this may take a little time to do.

#### Expected behavior
- As I'm not subscribing to the `mouseMoveZoom` handler, I would not normally expect any errors or data reported or related to this event.

**Comment by alexpreynolds** [comment]
Per Fritz's question, I observe this error when mousing over a gene annotation track.

**Comment by pkerpedjiev** [comment]
This is a an issue with the release. That bug is fixed but I think because we deleted v1.3.1 and re-released it the updates didn't make it into npm. Should be fixed in the next release.

**Comment by pkerpedjiev** [comment]
Fixed in v1.4.0
